Extensions/ArtifactEngine/Engine/artifactEngine.ts

```diff
@@ -15,6 +15,8 @@ import { TicketState } from '../Models/ticketState';
 export class ArtifactEngine {
     processItems(sourceProvider: models.IArtifactProvider, destProvider: models.IArtifactProvider, artifactEngineOptions?: ArtifactEngineOptions): Promise<models.ArtifactDownloadTicket[]> {
         var artifactDownloadTicketsPromise = new Promise<models.ArtifactDownloadTicket[]>((resolve, reject) => {
+            tl.setResourcePath(path.join(path.dirname(__dirname), 'lib.json'));
```

Comment:
lib.json seems generic name. will it cause conflicts?

Reply:
No, it wont cause any conflicts..